client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx

```diff
@@ -46,7 +46,10 @@ const BusinessPlanDetails = ( { selectedSite } ) => {
 };
 
 BusinessPlanDetails.propTypes = {
-	selectedSite: React.PropTypes.object.isRequired
+	selectedSite: React.PropTypes.oneOfType( [
+		React.PropTypes.bool,
```

Comment:
I'm not sure if these are accurate changes.

- This component isn't mounted until the receipt is loaded, and we assume the presence of a selected site when fetching the receipt in `CheckoutThankYou`.
- This component accesses properties of `selectedSite` without handling the case where `selectedSite` is false.